index.html

```diff
@@ -86,41 +78,169 @@ <h2 class="second-header">I am software developer</h2>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
           </div>
         </div>
       </div>
     </section>
     <section class="about-me">
       <div class="about-me-description">
-        <h2></h2>
-        <p class="intro-text"></p>
-        <!-- <button class="get-resume"></button> -->
+        <h2>About me</h2>
+        <p class="intro-text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="get-resume">get my Resume</button>
       </div>
 
```

Comment:
- [ ] Please the button should  follow the button styling: Active, Hover and Pressed (see design)
![image](https://user-images.githubusercontent.com/74744787/159044230-c492ba49-af97-4ab8-83b4-994c64ba6a3a.png)